device/dhcp/servers/v4/dhcpv4serverLeases.yaml

```diff
@@ -0,0 +1,76 @@
+components:
+  schemas:
+    DhcpServerV4.Leases:
+      description: >-
+        Configuration for DHCPv4 Leases.
+      type: object
+      x-status:
+        status: under_review
+      properties:
+        lease_time: 
+          description: >-
+            The Life Time length in seconds that is assigned to a lease if the requesting DHCP client does not specify a specific expiration time.
+          type: integer
+          format: uint32
+          minimum: 300
+          maximum: 30000000
+          default: 86400
+          x-field-uid: 1
+        start_pool_address:
+          description: >-
+            The IP address of the first lease pool.
+          type: string
+          format: ipv4
+          default: 100.1.1.1
+          x-field-uid: 2
+        prefix:
+          description: >-
+            The IPv4 network prefix length to be applied to the address. 
+          type: integer
+          format: uint32
+          default: 24
+          maximum: 32
+          x-field-uid: 3
+        pool_size:
+          description: >-
+            The total number of addresses in the pool.
+          type: integer
+          format: uint32
+          default: 1
+          minimum: 1
+          x-field-uid: 4
+        step:
+          description: >-
+            Increments the network address prefixes within a pool range  where multiple
```

Comment:
Something like "The increment value for the lease address within the lease pool."